src/Game/Managers/CommandManager.cs

```diff
@@ -38,7 +38,7 @@ public static void Initialize()
             Register("info", s =>
             {
                 if (!TargetManager.IsTargeting)
-                    TargetManager.SetTargeting(CursorTarget.SetTargetClientSide, 6983686, 0);
+                    TargetManager.SetTargeting(CursorTarget.SetTargetClientSide, CursorType.Target, TargetType.Neutral);
```

Comment:
:bulb: `6983686` was a magic number, so I externalized that. Open to suggestions on where this enumeration should live -- within the `TargetManager` may not be exactly the right place for it.